tests/test_units.py

```diff
@@ -144,3 +148,16 @@ def test_expression_printer(self, quantity_store):
         assert printer.doprint(a / b) == 'a[meter]/b[second]'
         assert printer.doprint(_2 * y) == '2.000000[dimensionless]*y[volt]'
         assert printer.doprint(sp.Derivative(a, b)) == 'Derivative(a[meter], b[second])'
+
+    def test_dimensionally_equivalent(self):
+        hodgkin_cellml = os.path.join(
```

Comment:
Is this our go-to test model? If so, perhaps we should have a pytest fixture for loading it

Reply:
https://github.com/ModellingWebLab/cellmlmanip/blob/develop/tests/test_parser.py#L10-L22

Reply:
I think this file has variables specifically for testing unit conversion: https://github.com/ModellingWebLab/cellmlmanip/blob/develop/tests/cellml_files/test_simple_odes.cellml

Reply:
Not sure right now there is only 1 test that uses it in there, but sure a fixture may be useful

Reply:
A shared fixture in `conftest.py` might make most sense?